### ISSUE TITLE
Description of keyboard shortcuts in video player is not accessible

### ISSUE THREAD

**Issue by dontcallmedom** [w3c/presentation-viewer#1]
The list of keyboard shortcuts exposed by JWPlayer is not accessible (e.g. no space between the labels and the actual key); I couldn't find a reported issue on the jwplayer repo, but the relevant code seems to be at https://github.com/jwplayer/jwplayer/blob/40b9b524fefe27b4f3537e050199a550791c4cc8/src/js/view/controls/templates/shortcuts-tooltip.ts#L11

(that said, our video streaming platform provider is looking at switching to a different player in the future, so not sure how much to invest in this in the short term)
